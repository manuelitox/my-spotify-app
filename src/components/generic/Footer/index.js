import React from 'react'

import StyledFooter from './styles'
import { ReactComponent as LoveIcon } from 'svgs/love.svg'
import { ReactComponent as SpotifyIcon } from 'svgs/spotify.svg'

const Footer = () => (
  <StyledFooter>
    <p>
      Powered by <SpotifyIcon />. Developed with <LoveIcon /> <a href="https://github.com/manuelitox" target="_blank" rel="noopener noreferrer">Manuel</a>.
    </p>
  </StyledFooter>
)

export default Footer