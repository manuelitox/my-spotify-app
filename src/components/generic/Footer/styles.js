import styled from 'styled-components'

const StyledFooter = styled.footer`
  color: ${ props => props.theme.textFooter };
  font-size: 1.2rem;
  font-weight: 500;
  text-align: center;
  aÂ {
    color: ${ props => props.theme.textFooter };
  }
`

export default StyledFooter