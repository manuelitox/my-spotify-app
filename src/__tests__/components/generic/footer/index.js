import React from 'react'
import { shallow } from 'enzyme'

import Footer from 'components/generic/Footer/'

describe('Component | Footer', () => {
  let component 
  beforeEach( () => {
    component = shallow(
      <Footer />
    )
  })
  it('should render a component', () => {
    expect(component).toMatchSnapshot()
  })
})