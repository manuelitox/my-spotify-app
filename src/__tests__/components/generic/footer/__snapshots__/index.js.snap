// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component | Footer should render a component 1`] = `
<styled.footer>
  <p>
    Powered by 
    <ReactComponent />
    . Developed with 
    <ReactComponent />
     
    <a
      href="https://github.com/manuelitox"
      rel="noopener noreferrer"
      target="_blank"
    >
      Manuel
    </a>
    .
  </p>
</styled.footer>
`;
